bin/jobs_functions.js

```diff
@@ -231,7 +231,6 @@ function reportForTestJob(testjobId, outputFolder, fileName, format) {
                     util.printErrorAndExit(`Status code: ${response.status}`);
                 } else {
                     util.output('Report created successfully');
-                    process.exit(0);
```

Comment:
Exiting the process as such ostensibly "cancels" the stream write operation resulting in 0 length report files.